index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <h1>Hello Microverse</h1>
+</body>
+</html>
```

Comment:
- [ ] Nice work including README file. However, please remove any default text that has nothing to do with your project, this does not include the heading. You should actually add some text to each of the headings as it relates to your project.
See image below - Please edit the getting started section as it relates to your project. 
![image](https://user-images.githubusercontent.com/84156714/191249768-bfc6025c-03f7-4862-941c-ca59cbf4e897.png)